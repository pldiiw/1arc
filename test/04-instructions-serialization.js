'use strict';

const serializeInstructions = require('interpreter').serializeInstructions;

const goodProgram = '611181107111C1111111A111F11ED111';
console.log(serializeInstructions(goodProgram));

const secondGoodProgram = '811EB111211100EE00E0F129';
console.log(serializeInstructions(secondGoodProgram));

const badProgram = goodProgram + 'F132';
console.log(serializeInstructions(badProgram));

const secondBadProgram = secondGoodProgram + 'FG55';
console.log(serializeInstructions(secondBadProgram));
