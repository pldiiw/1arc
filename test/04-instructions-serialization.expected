611181107111C1111111A111F11ED111
811EB111211100EE00E0F129
null
null
